README.md

```diff
@@ -39,21 +39,13 @@ Use:
 dart format lib/ test/ -l 120
 ```
 
-In Android Studio, set the line length using Preferences -> Editor -> Code Style -> Dart -> Line length, set it to 120.  Enable auto-format with Preferences -> Languages & Frameworks -> Flutter -> Format code on save.
+In Android Studio, set the line length using Preferences → Editor → Code Style → Dart → Line length, set it to 120.  Enable auto-format with Preferences → Languages & Frameworks → Flutter → Format code on save.
 
+# Prerelease
 
-# Release
-
-Update `version` in `pubspec.yaml` to reflect this release, then
-
-## Android
+Push a git tag `v#.#.#-##`, e.g. `v0.5.2-0`, and `.github/release.yml` will build a draft release and publish it to iOS TestFlight and Android internal track.
 
-`flutter build appbundle`
 
-This will create an android app bundle at `build/app/outputs/bundle/release/`
-
-Upload the android bundle to the google play store https://play.google.com/apps/publish
-
-## iOS
+# Release
 
-In xcode, Release -> Archive then follow the directions to upload to the app store. If you have issues, https://flutter.dev/docs/deployment/ios#create-a-build-archive
\ No newline at end of file
+We manually promote a prerelease build from TestFlight and Android internal track to the public app stores, and then mark the associated draft release as published, removing the `-##` from it.
```

Comment:
> removing the `-##` from it.

To be specific: we are creating a new tag in the format `v#.#.#` by removing the `-##`, updating the title of the release to `Release v#.#.#`, and ideally we would also populate the changes in this release taken from the CHANGELOG.md

For the CHANGELOG.md, we lost our flow there and would be great to get back into the habit of updating it.

Reply:
This is all done by editing the draft release in GitHub ui and not by publishing a new tag to the same commit, right?

Reply:
Updated!